build_tools/waf_dynamo.py

```diff
@@ -1881,8 +1881,10 @@ def detect(conf):
         conf.env['LIB_OPENGL'] = ['EGL', 'GLESv1_CM', 'GLESv2']
     elif platform in ('win32', 'x86_64-win32'):
         conf.env['LINKFLAGS_OPENGL'] = ['opengl32.lib', 'glu32.lib']
-    elif platform in ('x86_64-linux','arm64-linux'):
+    elif platform in ('x86_64-linux'):
         conf.env['LIB_OPENGL'] = ['GL', 'GLU']
+    elif platform in ('arm64-linux'):
+        conf.env['LIB_OPENGL'] = ['EGL', 'GLESv1_CM', 'GLESv2']
```

Comment:
My main concern is if there will be an actual "desktop" target using this.
For better support in the future, I think the these options should be chosen using an app manifest approach (i.e. choosing the graphics backend: opengl, gles, webgpu etc).

Question is, can we go a head with this, and not make it breaking change if we want to support arm64-linux desktop targets?
I wonder if it's not better to default to OpenGL, and leave the custom backend selection for the app manifest.
(we've also wanted to move to Vulkan as default which just highlights the point)